RELEASE_GUIDE.md

```diff
@@ -44,48 +67,142 @@ considering what effects this may have:
+
+After that, the respective packages on Hackage and NPM will be uploaded/updated.
+
+### Set up a local branch for the PR
+
+```bash
+cd purescript
+git fetch upstream
+git checkout upstream/master
+git switch -c prepareNextRelease
+```
+
+### Make last changes and submit the PR
+
+```bash
+# 1. Regenerate the LICENSE file and commit it
+# See `./license-generator/` for details)
+make license-generator
+git add LICENSE
+git commit -m "Regenerated LICENSE file"
+
+# 2. Edit the changelog
+  # 2a. Create a new "Unreleased" section
+  # 2b. Rename the "Unreleased" section to
+  #   the version you will be releasing
+  # 2c. Edit the new version's content
+  #   until satisfied (e.g. grammer,
+  #   spelling, clarity, etc.)
+
+EDITOR="nano" # or your preferred editor
+$EDITOR CHANGELOG.md
+git add CHANGELOG.md
+git commit -m "Added new Unreleased section; finalized new version's section"
+
+# 3. Bump the versions in each file below:
+
+EDITOR="nano" # or your preferred editor
 
-  - The `version` field in `purescript.cabal`
+  # 3a. If it has changed since the
+  #   last release, bump the version
+  #   The new version should be based on the PVP,
+  #   according to what changed since the previous release, #   not on the actual compiler version.
+  #   See https://pvp.haskell.org/
+$EDITOR lib/purescript-cst/purescript-cst.cabal
 
-  - The `prerelease` field in `app/Version.hs`, if updating the prerelease
-    field
+  # 3b. Bump the version
+  # 3c. If it changed, update the `purescript-cst`
+  #   version to match the new one
+$EDITOR purescript.cabal
 
-  - The `version` field in `npm-package/package.json`
+  # 3d. If making a pre-release,
+  # change the `prerelease` string.
+  # Otherwise, it should be "" for all
+  # other releases.
+$EDITOR app/Version.hs
 
-  - The version to install in the `postinstall` script in `package.json`
+  # 3e. Update the table
+$EDITOR lib/purescript-cst/README.md
 
-  - If `purescript-cst` has changed at all since the last release:
+  # 3f. Change the "version" field
+  # 3g. In the `postinstall` script,
+  #   change the version.
+$EDITOR npm-package/package.json
 
-      - The `version` field in `lib/purescript-cst/purescript-cst.cabal` (note
-        that the new version should be based on the PVP, according to what
-        changed since the previous release, and not on the actual compiler
-        version)
+git add \
+  lib/purescript-cst/purescript-cst.cabal \
+  lib/purescript-cst/README.md \
+  purescript.cabal \
+  app/Version.hs \
+  npm-package/package.json
+git commit -m "Bumped versions for upcoming release"
 
-      - The versions table in `lib/purescript-cst/README.md`,
+# 4. Create the PR either manually or
+#   via the `gh` CLI tool
+gh pr create
+# a. Make the `base` repo `purescript`
+# b. Push your branch to your fork
+# c. Continue on the web
 
-      - The version bounds for `purescript-cst` in `purescript.cabal`
+# 5. Push any other changes to the PR
```

Comment:
I am not sure that the changes to this section are an improvement. They add quite a bit of noise, and explain how to do basic tasks like creating git branches, opening files in  their editor, or submitting PRs, but we should assume that a maintainer creating a release knows how to do these things. I think the extra noise is obscuring the important information here, which is which parts of which files need updating.

Reply:
Yeah... I have to agree with you here. It also doesn't account for people who are using editors that can easily navigate to such files.